package.json

```diff
@@ -8,7 +8,7 @@
     "build": "./script/build.sh",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
     "build:compare": "node script/content-build-compare.js && diff --color websiteContentBuildHash.txt standaloneContentBuildHash.txt",
-    "build:content": "node --max-old-space-size=8192 script/build-content.js",
+    "build:content": "node --max-old-space-size=10240 script/build-content.js",
```

Comment:
This won't be needed if we merge #15601.

Reply:
I haven't seen recent PR's com across this issue. It also looks like your [pipeline](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/expand_vamc_tests/8/pipeline) is still failing. Have you tried re running again?

Reply:
That's probably because the CI environment uses a cache of the drupal data.

When that cache is updated, I expect all of the PR's will start failing.

Reply:
You'll probably need to increase the `max-old-space-size` in `jenkins/common.groovy` as well.

Reply:
Looks that way, yes. Hopefully once #15601 is merged we can set it back if need be.